libs/bulldozer/application/utils/services/code-editor-settings/src/lib/code-editor-settings.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
```

Comment:
Is the code-editor-settings library still needed?

Reply:
No its not, due that we now don hide the code editor, its not necessary. I'll delete it.